tsconfig.json

```diff
@@ -7,6 +7,7 @@
 		"target": "ES2015",
 		"moduleResolution": "node",
 		"resolveJsonModule": true,
-		"allowSyntheticDefaultImports": true
+		"allowSyntheticDefaultImports": true,
+		"skipLibCheck": true
```

Comment:
Bitte hier noch einen entsprechenden Kommentar ergänzen, warum das geskippt wird. :3

Reply:
Das ist leider JSON. JSON kennt (afaik) keine Kommentare 🤷‍♀️
Also der Grund dafür ist, dass ich gegen die Fehler in den importierten Libs nichts tun kann (oder möchte)...

Reply:
Okay, ich wurde darüber informiert, dass _diese_ JSON File wohl ausnahmsweise Kommentare kann.
Mir wurde auch geraten, die einfach mal neu zu generieren, weil da wohl ein paar gute Defaults drin stehen. Da kümmer ich mich morgen drum.

Reply:
Von der neu generierten tsconfig war ich nicht so überzeugt. Habe aber rausgefunden, dass sich playwright an alten node types aufhing.
Jetzt ist also das ursprünglich zugrundelegende Problem gelöst und wir können auf das skipLibCheck verzichten :)